templates/http/default.conf.j2

```diff
@@ -12,8 +12,39 @@ upstream {{ item.value.upstreams[upstream].name }} {
     sticky cookie srv_id expires=1h  path=/;
 {% endif %}
 }
+{% endfor %}
+{% endif %}
 
+{% if (item.value.reverse_proxy_enable is defined) and item.value.reverse_proxy_enable %}
```

Comment:
I've seen you've added a `reverse_proxy_enable` variable. Ideally I'd prefer to use a conditional check such as `reverse_proxy is defined` instead.

Reply:
Done